lib/pages/AppConnectorConfig/AppConnectorConfig.tsx

```diff
@@ -142,30 +140,24 @@ const AppConnectorConfig: React.FC = () => {
   }
 
   const saveSubscription = (event) => {
```

Comment:
Upon testing this function, I came to find that `event` and `event.preventDefault()` are not necessary.

If that's okay, I suggest removing both.